packages/framer-motion/src/render/html/HTMLVisualElement.ts

```diff
@@ -11,87 +9,52 @@ import { scrapeMotionValuesFromProps } from "./utils/scrape-motion-values"
 import { renderHTML } from "./utils/render"
 import { getDefaultValueType } from "../dom/value-types/defaults"
 import { measureViewportBox } from "../../projection/utils/measure"
+import { VisualElement } from "../VisualElement"
+import { MotionProps, MotionStyle } from "../../motion/types"
+import { MotionValue } from "../../value"
+import { MotionConfigProps } from "../../components/MotionConfig"
+import { Box } from "../../projection/geometry/types"
+import { IProjectionNode, ResolvedValues, TargetAndTransition } from "../.."
 
 export function getComputedStyle(element: HTMLElement) {
     return window.getComputedStyle(element)
 }
 
-export const htmlConfig: VisualElementConfig<
-    HTMLElement,
-    HTMLRenderState,
-    DOMVisualElementOptions
-> = {
-    treeType: "dom",
-
-    readValueFromInstance(domElement, key) {
-        if (transformProps.has(key)) {
-            const defaultType = getDefaultValueType(key)
-            return defaultType ? defaultType.default || 0 : 0
-        } else {
-            const computedStyle = getComputedStyle(domElement)
-            const value =
-                (isCSSVariable(key)
-                    ? computedStyle.getPropertyValue(key)
-                    : computedStyle[key]) || 0
-
-            return typeof value === "string" ? value.trim() : value
-        }
-    },
-
-    sortNodePosition(a, b) {
+export abstract class DOMVisualElement<
```

Comment:
Can we put this in it's own file?

Reply:
Ah yeah I will do - this was actually messing with VSC's parsing of the second class too